docs/how-to-guides/feathr-configuration-and-env.md

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
Which file is this going to replace? Seems another setup wiki.

Reply:
yes this is a separate doc/wiki.